spec/factories/users.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :user do
     etag { SecureRandom.hex }
-    sequence(:username) { |n| "#{Faker::Internet.user_name}_#{n}" }
+    sequence(:username) { |n| "#{Faker::Name.first_name}_#{n}" }
```

Comment:
`Faker::Name.first_name` includes names with apostrophes, like D'Angelo, which can lead to transient test failures.  :(